projects/goharbor/harbor/helm/sedfile.template

```diff
@@ -1,15 +1,6 @@
 s/^version: 1.8.1$/version: ${IMAGE_TAG}-helm/
 s/^name: harbor$/name: harbor-helm/
-s/digest: {{ NGINX_IMAGE_DIGEST }}$/digest: ${NGINX_IMAGE_DIGEST}/
-s/digest: {{ PORTAL_IMAGE_DIGEST }}$/digest: ${PORTAL_IMAGE_DIGEST}/
-s/digest: {{ CORE_IMAGE_DIGEST }}$/digest: ${CORE_IMAGE_DIGEST}/
-s/digest: {{ JOBSERVICE_IMAGE_DIGEST }}$/digest: ${JOBSERVICE_IMAGE_DIGEST}/
-s/digest: {{ REGISTRY_IMAGE_DIGEST }}$/digest: ${REGISTRY_IMAGE_DIGEST}/
-s/digest: {{ REGISTRYCTL_IMAGE_DIGEST }}$/digest: ${REGISTRYCTL_IMAGE_DIGEST}/
-s/digest: {{ CHARTMUSEUM_IMAGE_DIGEST }}$/digest: ${CHARTMUSEUM_IMAGE_DIGEST}/
-s/digest: {{ TRIVY_ADAPTER_IMAGE_DIGEST }}$/digest: ${TRIVY_ADAPTER_IMAGE_DIGEST}/
-s/digest: {{ NOTARY_SERVER_IMAGE_DIGEST }}$/digest: ${NOTARY_SERVER_IMAGE_DIGEST}/
-s/digest: {{ NOTARY_SIGNER_IMAGE_DIGEST }}$/digest: ${NOTARY_SIGNER_IMAGE_DIGEST}/
-s/digest: {{ DATABASE_IMAGE_DIGEST }}$/digest: ${DATABASE_IMAGE_DIGEST}/
-s/digest: {{ REDIS_IMAGE_DIGEST }}$/digest: ${REDIS_IMAGE_DIGEST}/
-s/digest: {{ EXPORTER_IMAGE_DIGEST }}$/digest: ${EXPORTER_IMAGE_DIGEST}/
\ No newline at end of file
+s,{{harbor/harbor-chartmuseum}},sha256:f5ac705ce6173725ac2f9da9891d13f1000e78caddbeca36e8c1c5b12e0f86fc,g
```

Comment:
What are these shas?

Reply:
They are bogus shas for now. They won't work, but those images are not used.